### ISSUE TITLE
operation-costs fails with KeyError: "['GFA_m2'] not in index"

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#1183]
When the operation-costs on the default scenario (after running data-helper, demand, embodied-energy, emissions and mobility scripts), the script fails with the following error:

```
λ cea operation-costs
Running operation-costs with scenario = C:\reference-case-open\baseline
Traceback (most recent call last):
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\Scripts\cea-script.py", line 11, in <module>
    load_entry_point('cityenergyanalyst', 'console_scripts', 'cea')()
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\interfaces\cli\cli.py", line 33, in main
    script_module.main(config)
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\analysis\lca\operation_costs.py", line 95, in main
    operation_costs(locator=locator, config=config)
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\analysis\lca\operation_costs.py", line 87, in operation_costs
    result[['Name']+fields_to_plot].to_csv(locator.get_costs_operation_file(), index=False, float_format='%.2f')
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\core\frame.py", line 1958, in __getitem__
    return self._getitem_array(key)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\core\frame.py", line 2002, in _getitem_array
    indexer = self.loc._convert_to_indexer(key, axis=1)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\cea\lib\site-packages\pandas\core\indexing.py", line 1231, in _convert_to_indexer
    raise KeyError('%s not in index' % objarr[mask])
KeyError: "['GFA_m2'] not in index"
```

Note that the default configuration file was used (i.e., the `cea.config` file was deleted from the home folder before running the scripts.

**Comment by daren-thomas** [comment]
The offending line is this one: (operation_costs.py, line 86)

```
result = heating.merge(dhw, on='Name').merge(cooling, on='Name').merge(electricity, on='Name')
```

When merging heating, cooling and electricity, the field names get renamed according to the documentation to  `['GFA_m2_x', 'GFA_m2_y', 'GFA_m2_x', 'GFA_m2_y']` - this happens with all the other merged fields too!

Fixed in ddb18e1dcdd9cb77cb8a3f18c1720896f528d062

**Comment by daren-thomas** [comment]
Next problem: 
```
You are trying to put colors in one variable that is not recognized by CEA, check the list of variables,recognized in the file color_code.py, We are randomly selecting a new color pallete
```
That message shows up a dozen times. Why?
